src/catalog/catalog_structs.cpp

```diff
@@ -69,6 +69,15 @@ Property TableSchema::getProperty(property_id_t propertyID) const {
         "Table: %s doesn't have a property with propertyID=%d.", tableName.c_str(), propertyID));
 }
 
+void TableSchema::renameProperty(property_id_t propertyID, string newName) {
```

Comment:
I guess your IDE will suggest you to use `const string&`

Reply:
changed to `const string&`